src/core/server/base.js

```diff
@@ -13,19 +13,21 @@ import { match } from 'react-router';
 import { ReduxAsyncConnect, loadOnServer } from 'redux-async-connect';
 
 import WebpackIsomorphicTools from 'webpack-isomorphic-tools';
-import WebpackIsomorphicToolsConfig from 'config/webpack-isomorphic-tools';
+import WebpackIsomorphicToolsConfig from 'webpack-isomorphic-tools-config';
 
 import config from 'config';
 import { setJWT } from 'core/actions';
 
+const env = config.util.getEnv('NODE_ENV');
+const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
```

Comment:
Ah I see what you mean, the `start` targets used to default to production prior to there needing to be a `stage`, `dev` and `production` `NODE_ENV`. We should probably make `start:[x]` targets complain if you don't specify a deployable `NODE_ENV`.